sklearn/utils/__init__.py

```diff
@@ -525,6 +530,57 @@ def indices_to_mask(indices, mask_length):
     return mask
 
 
+def message_with_time(source, message, time):
+    """Create one line message for logging purposes
+
+    Parameters
+    ----------
+    source : str
+        String indicating the source or the reference of the message
+
+    message : str
+        Short message
+
+    time : int
+        Time in seconds
+    """
+    start_message = '[%s]' % (source,)
+
+    # from joblib.logger.short_format_time without the Windows -.1s adjustment
+    if time > 60:
+        time_str = "%4.1fmin" % (time / 60)
+    else:
+        time_str = " %5.1fs" % (time)
+    end_message = "%s, elapsed=%s" % (message, time_str)
+    dots_len = (68 - len(start_message) - len(end_message))
```

Comment:
I feel like ``"{:<68}" should work here, but not sure how to do it with dots? We could replace the spaces by dots afterwards but not sure that's actually simpler?

Reply:
After going through a few iterations of this log message where the dots are in the center, I believe the current way is the most easy to follow.